exotic/api/plotting.py

```diff
@@ -118,7 +118,7 @@ def plot_image(filename, save=False, bg_min=60, bg_max=99):
 
     # create a figure with text on mouse hover\
     print("Saturated pixels are marked with red. These are pixels which have exceeded the maximum value for brightness, and are thus not suitable for use as comparison stars.")
-    fig = figure(tooltips=[("x", "$x"), ("y", "$y"), ("value", "@image")], plot_width=800, plot_height=800,
+    fig = figure(tooltips=[("x", "$x"), ("y", "$y"), ("value", "@image")], width=800, height=800,

```

Comment:
@jpl-jengelke - should this be a try/except instead to ensure backwards compatibility with the previous version of bokeh?

Reply:
Well, I thought no. Since it will be tagged as 4.0.1, we just need to make sure that it is only used to accompany a new Colab notebook. So releasing an upgraded notebook would be the solution. 

In other words, it's isolated from the 3.3.0 release, so ideally shouldn't be a problem. Moreover, the modules need to be upgraded so supporting old modules wouldn't comport with security recommendations.